### PR TITLE
KEYCLOAK-10747 Explicit Proof Key for Code Exchange Activation Settings

### PR DESCRIPTION
**Pull request by tnorimat**
This is the documentation for https://github.com/keycloak/keycloak/pull/6135 .

### CODE REVIEW

**Review by mposolda — DISMISSED** [review]
@tnorimat Thanks!

Adding @andymunro as a reviewer of the PR to ensure that everything is good from the perspective of the documentation team

File: server_admin/topics/clients/client-oidc.adoc [hunk 1]
```diff
@@ -149,6 +149,21 @@ WARNING:
 None of the keycloak client adapters currently support holder-of-key token verification.
 Instead, keycloak adapters currently treat access and refresh tokens as bearer tokens.
 
+[[_proof-key-for-code-exchange]]
+*Proof Key for Code Exchange (PKCE)*
+
+PKCE prevents an attacker who steals an authorization code issued to a legitimate client from receiving the tokens in return to their stealing authorization code.
```
Comment (andymunro) [inline server_admin/topics/clients/client-oidc.adoc]: I suggest wording it this way:
When an attacker steals an authorization code that was issued to a legitimate client, PKCE prevents the attacker from receiving the tokens that apply to that code.

File: server_admin/topics/clients/client-oidc.adoc [hunk 2]
```diff
@@ -149,6 +149,21 @@ WARNING:
...
+
+The administrator can select the following three options:
+
+*Proof Key for Code Exchange Code Challenge Method*
+
+* (blank) : {project_name} does not apply PKCE unless the client sends PKCE's parameters appropriately to keycloak's authorization endpoint. It is default setting.
```
Comment (andymunro) [inline server_admin/topics/clients/client-oidc.adoc]: I suggest:  It is the default setting.

File: server_admin/topics/clients/client-oidc.adoc (hunk 1 above)
Comment (tnorimat) [reply server_admin/topics/clients/client-oidc.adoc]: Thank you. I'll revise the sentence as you suggested.

File: server_admin/topics/clients/client-oidc.adoc (hunk 2 above)
Comment (tnorimat) [reply server_admin/topics/clients/client-oidc.adoc]: Yes, I'll revise the sentence as you suggested.

**Comment by tnorimat** [comment]
The build failed but not relevant to this pull request.

The error log shows that :

checkExternalLinks(org.keycloak.documentation.test.SecuringAppsTest)  Time elapsed: 26.046 sec  <<< FAILURE!
* https://developer.apple.com/ios/universal-links/ (invalid status code 403)

It seems that javascript-adapter.adoc line:200 cause it.

**Review by mposolda — APPROVED** [review]
@tnorimat Thanks for incorporating feedback.
@andymunro Build passed now and @tnorimat added changes. Do you have a chance to re-review please? Thanks

File: server_admin/topics/clients/client-oidc.adoc (hunk 2 above)
Comment (andymunro) [reply server_admin/topics/clients/client-oidc.adoc]: Sorry, Takashi, but when I click View Changes, the message is that "it cannot find the content anywhere."  I am still learning how to view changes in GitHub.

**Review by andymunro — APPROVED** [review]

File: server_admin/topics/clients/client-oidc.adoc (hunk 2 above)
Comment (andymunro) [reply server_admin/topics/clients/client-oidc.adoc]: Okay, I found out where to see the changes.  Looks good. Thanks.

**Review by andymunro — APPROVED** [review]

**Pull request merged**